.github/workflows/publish.yml

```diff
@@ -0,0 +1,36 @@
+# ---------------------------------------------------------------
+# workflow: publish
+#
+# Perform on all release:create events
+#
+# Signs and published artifacts to Maven Central
+#
+on:
+  release:
+    types: [created]
+
+name: publish
+
+jobs:
+  publish:
+    name: 'Publish Release'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'corretto'
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+      - name: Publish Release
+        run: |
+          export ORG_GRADLE_PROJECT_signingKey=$(echo -n "$GPG_SIGNING_KEY_BASE64" | base64 -d)
+          ./gradlew publishToMavenLocal
```

Comment:
This is just doing a local publish to test the signing. Will change this once the publish is configured.